### DISCUSSION TITLE
Reminders "whichever comes later"

### DISCUSSION THREAD

**Discussion by augmentedagent** [hargata/lubelog#333] [Ideas]
It would be nice to have the option to select "whichever comes later" in addition to "whichever comes first" on reminders. For example on oil changes, one might put in the date field 6 months from the current date, and in the future miles field 6000 miles from current mileage, but not want to be reminded until the later of the 2 items has happened.

**Comment by chirmstream** [comment]
Personally, I'm having a hard time to imagine when I would want the reminder to be setup in your example.  If the two events are to happen within a similar time frame, then it doesn't matter and you could use mileage or time for your reminder.  If they are to happen at different times, then create your reminder using only the event that will happen second.  You would need to be familiar with your vehicles usage, but I  don't see that being an issue for the type of person who is already tracking and logging this stuff.

**Reply by augmentedagent** [reply]
This would only be useful in the case of them happening at different times. As far as being familiar with usage, sometimes there are spikes that the user (me) doesn't know about due to not being the driver of many of the vehicles in the fleet.

**Reply by chirmstream** [reply]
Have multiple vehicles in your fleet, where you are not the primary driver makes sense.
